recipes-bsp/u-boot/README.md

```diff
@@ -59,6 +59,15 @@ apt-get install flashrom
 
 Attach the grabber to the flash chip, then run the following on the host machine:
 
+While the the size of the uboot file doesn't match the flash chip's size (16777216 B)
+execute this command:
+
+```shell
+truncate -s 16M iot2050-image-boot.bin
+```
+
+Then, upgrade  uboot:
```

Comment:
Strictly spoken, the update will not only be U-Boot but the complete firmware. I would write: "Then update the firmware:"

Reply:
Thanks a lot, I will update the md to be more accurate